ostrich_swmm/inject.py

```diff
@@ -110,6 +113,8 @@ def inject_parameters_into_input(input_parameters, input_template):
 
     Raises:
         ConfigException: The configuration is invalid.
+    Returns:
+        excess_rb for input into extract csv
```

Comment:
It doesn't look like this function returns anything, so these lines should be removed from the function's documentation.